package.json

```diff
@@ -45,7 +45,7 @@
     "lint-staged": "^10.1.3",
     "lodash.startcase": "^4.4.0",
     "prettier": "1.19.1",
-    "prettier-plugin-solidity": "^1.0.0-alpha.59",
+    "prettier-plugin-solidity": "1.0.0-alpha.59",
```

Comment:
I added this to lock the prettier version as the latest version introduced a lot of changes. this should come in as a separate PR, as done here https://github.com/UMAprotocol/protocol/pull/2209